two-factor.php

```diff
@@ -6,41 +6,15 @@
  * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
  */
 
-add_filter( 'two_factor_providers', function( $p ) {
-	if ( wpcom_vip_have_twilio_keys() ) {
-		$p['Two_Factor_SMS'] = __DIR__ . '/wpcom-vip-two-factor/sms-provider.php';
-	}
-
-	// https://github.com/georgestephanis/two-factor/issues/78
-	unset( $p[ 'Two_Factor_FIDO_U2F' ] );
-	unset( $p[ 'Two_Factor_Dummy' ] );
-	return $p;
-} );
-
-function wpcom_vip_have_twilio_keys() {
-	return defined( 'TWILIO_SID' ) && ! empty( TWILIO_SID )
-		&& defined( 'TWILIO_SECRET' ) && ! empty( TWILIO_SECRET );
-}
-
-function wpcom_vip_is_jetpack_sso_enabled() {
-	return class_exists( 'Jetpack' )
-		&& Jetpack::is_active()
-		&& Jetpack::is_module_active( 'sso' )
-		// SSO does not work for staging sites
-		&& ! Jetpack::is_staging_site();
-}
+// Custom list of providers
+require_once __DIR__ . '/wpcom-vip-two-factor/set-providers.php';
 
 function wpcom_vip_force_two_factor() {
```

Comment:
Minot: thoughts on renaming this from `wpcom_vip_force_two_factor` to `wpcom_vip_is_two_factor_forced`?

This always trips me up when reading this file; especially since the function isn't forcing anything, but rather checking a state.

Reply:
Done in a831d3b